package.json

```diff
@@ -60,8 +60,8 @@
     }
   },
   "dependencies": {
-    "@octokit/rest": "^16.28.7",
-    "auth0": "^2.19.0",
+    "@octokit/rest": "^16.33.1",
+    "auth0": "^2.20.0",
```

Comment:
Check if all dependencies are in use and in place (dotenv is a devDepency), could be in another PR